src/support/assertRender.ts

```diff
@@ -37,7 +37,7 @@ function throwAssertionError(actual: any, expected: any, message?: string): neve
  */
 const defaultDiffOptions: DiffOptions = {
 	allowFunctionValues: true,
-	ignoreProperties: [ 'bind', 'defaultRegistry' ]
+	ignoreProperties: [ 'bind', 'afterCreate', 'afterUpdate' ]
```

Comment:
Need to look at why these are being added when they weren't expected before the update.